### PR TITLE
Update 03-premium.md and 04-bonds.md

### PR DESCRIPTION
**Pull request by okada-blue**
**Update 03-premium.md**
Update 'summary' portion to be more concise and straight forward in describing how premiums are to be viewed in the order book (inspired by a question on TG). I felt some of the verbiage in the introduction was redundant and could be removed for simplicity.

**Update 04-bonds.md**
Proposed rough draft for 'Bonds' documentation. I'm not certain I have captured every scenario in which bonds can be lost. Should I even bother mentioning bondless takers if it's not an option right now?

### CODE REVIEW

File: docs/_pages/docs/03-understand/04-bonds.md [hunk 1]
```diff
@@ -2,10 +2,54 @@
...
+
+After the trade is completed and both robots are satisfied, the maker and taker bonds are relinquished. Technically, the locked bond never left your wallet; but take caution, if you fail to follow the contract obligations by trying to cheat or cancelling unilaterally, you will forfeit your fidelity bond.
+
+Your wallet may take a while for funds to show as relinquished on your account balance. Some wallets have difficulty with recognizing the Lightning hold invoice as a temporary hold on your funds.
+
+If the issue persists, please reach out to the RoboSats Telegram group; but beware of scammers that may directly contact you and impersonate RoboSats staff! RoboSats staff will never directly contact you first. See [Contribute > Code > Communication Channels](https://learn.robosats.com/contribute/code/#communication-channels) for available Telegram groups. 
+
+## **Losing Your Bond**
```
Comment (Reckless-Satoshi) [inline docs/_pages/docs/03-understand/04-bonds.md:37]: We could add some bullet point summarizing the conditions in which you lose the bond (basically these 4):
- Trying to cheat or deceive your peer (i.e., losing a dispute)
- As a buyer: failing to submit a payout invoice in time.
- As a seller: failing to post the trade collateral in time.
- As a seller: failing to release the escrow Satoshis to the buyer by confirming "fiat received".

File: docs/_pages/docs/03-understand/04-bonds.md [hunk 2]
```diff
@@ -2,10 +2,54 @@
...
   title: '<img id="side-icon-verybig" src="/assets/vector/ticket-simple.svg"/>Bonds'
   nav: docs
 src: "_pages/docs/03-understand/04-bonds.md"
 ---
 
-{% include wip %}
\ No newline at end of file
+The fidelity bond is a small deposit that the user "locks" which will be relinquished after the trade is completed; however, users can lose their bond if they fail to follow the obligations of the contract.
+
+The RoboSats trade pipeline utilizes fidelity bonds to incentivize both the order maker and taker to play by the rules and not cheat their fellow robot. More specifically, the bonds are hold invoices using the Lightning Network.
```
Comment (Reckless-Satoshi) [inline docs/_pages/docs/03-understand/04-bonds.md:15]: I very much like these explanations! Hold invoices are cool and is the tech that makes RoboSats possible. Maybe we can add a link to some in depth technical document of how hodl invoices work?

**Review by Reckless-Satoshi — COMMENTED** [review]
This is looking excellent! Thanks once a gain for helping complete the docs. Submit another 20K invoice with your next commit :rocket:

File: docs/_pages/docs/03-understand/04-bonds.md [hunk 3]
```diff
@@ -2,10 +2,54 @@
...
+
+If the time limit for submitting the invoice (buyer) or locking the escrow (seller) runs out, then the order will expire and the robot who did not hold up to their end of the deal will lose the bond.
+
+Therefore, don't forget about your order because once a robot takes it and locks their fidelity bond, you could lose your bond since the timer might expire. Take care to remember your order and back up your robot's unique token!
+
+Due to the time limits involved in the order process, it is recommended to use instant fiat payment methods which help reduce the chances of losing your bond. Refer to [Best Practices > Payment Methods](https://learn.robosats.com/docs/payment-methods/) for additional information.
+
+Opening a dispute just to cancel an order is not recommended because one of the two traders will lose their fidelity bond, barring exceptional cases that are up to the discretion of the RoboSats staff. 
+
+If RoboSats suddenly vanished or was shutdown, bonds are automatically relinquished since they technically never left your wallet.
```
[the remark this answers is not in the document]
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:49]: I agree, I think "relinquished" is not the appropriate term here. I will update accordingly

It's almost funny.. at time of writing I was trying to think of the best antonym for "locked" and I came up with "relinquished". But in more accurate terms it is simply "unlocked".... lol

File: docs/_pages/docs/03-understand/04-bonds.md [hunk 4]
```diff
@@ -2,10 +2,54 @@
...
+
+## **Don't Have Any Bitcoin for Bonds?**
+
+Because the bonds require a Lightning hold invoice, what are you to do if you have no bitcoin to begin with? Even though the bond is typically just 3% of your total trade amount, this presents a real barrier to using RoboSats for the first time if your sat stack is non-existent.
```
[the remark this answers is not in the document]
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:53]: I've not used many of the mentioned apps, but have heard positive experiences from other uses. I think it worth including just to point people in the right direction and not leave them at a dead-end. I will update accordingly, only including the apps I've heard highly recommended. We can always update to include more/better apps in future!

File: docs/_pages/docs/03-understand/04-bonds.md [hunk 5]
```diff
@@ -2,10 +2,54 @@
...
+* **Taker**: Browse the order book and find an order to your liking. Simply select the "Take Order" option and then use the following QR code found in the "Contract Box" with your Lightning wallet to lock the indicated amount of sats for your fidelity bond. *Note: Be prepared with your wallet beforehand because the order box expires in four minutes. If you do not proceed, the taken order is made public again.*
+
+After the trade is completed and both robots are satisfied, the maker and taker bonds are relinquished. Technically, the locked bond never left your wallet; but take caution, if you fail to follow the contract obligations by trying to cheat or cancelling unilaterally, you will forfeit your fidelity bond.
```
[the remark this answers is not in the document]
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:31]: Agreed, updated accordingly

File: docs/_pages/docs/03-understand/04-bonds.md [hunk 6]
```diff
@@ -2,10 +2,54 @@
...
+
+Your wallet may take a while for funds to show as relinquished on your account balance. Some wallets have difficulty with recognizing the Lightning hold invoice as a temporary hold on your funds.
+
+If the issue persists, please reach out to the RoboSats Telegram group; but beware of scammers that may directly contact you and impersonate RoboSats staff! RoboSats staff will never directly contact you first. See [Contribute > Code > Communication Channels](https://learn.robosats.com/contribute/code/#communication-channels) for available Telegram groups. 
+
+## **Losing Your Bond**
+
+If you received fiat but neglect to click "Confirm Fiat Received" on your end, then you risk losing your bond since a dispute is automatically opened and the RoboSats staff will find you failed to follow the rules of the contract.
+
+If the time limit for submitting the invoice (buyer) or locking the escrow (seller) runs out, then the order will expire and the robot who did not hold up to their end of the deal will lose the bond.
```
[the remark this answers is not in the document]
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:41]: Great point, updated accordingly

File: docs/_pages/docs/03-understand/04-bonds.md [hunk 7]
```diff
@@ -2,10 +2,54 @@
...
+
+*Note: The option allowing "Bondless Takers" is in the works but not available at the moment.*
+
+## **How to Lock a Bond**
+
+First, refer to [Understand > Wallets](https://learn.robosats.com/docs/wallets/) for compatible Lightning wallets that will help make using RoboSats a smoother experience. Depending on the wallet, the invoice might show as a payment that is in transit, frozen, or even appearing to fail. Check the wallet compatability list!
+
+Read the relevant guide depending on if you are making or taking the order:
+* **Maker**: Select "Make Order" and modify the order conditions to your liking. The order can be customized to require a fidelity bond other than the default 3% of the total trade amount, ranging anywhere from 2% to 15%. Once complete, confirm with "Create Order" and then use the following QR code found in the "Contract Box" with your Lightning wallet to lock the indicated amount of sats for your fidelity bond. *Note: Be prepared with your wallet beforehand because the order box expires in ten minutes.*
```
[the remark this answers is not in the document]
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:28]: I squeezed it in there, I think it fits/reads fine in that section of the doc!

File: docs/_pages/docs/03-understand/04-bonds.md [hunk 8]
```diff
@@ -2,10 +2,54 @@
 layout: single
 title: Maker and Taker Bonds
 permalink: /docs/bonds/
+toc: true
```
[the remark this answers is not in the document]
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:5]: I'm learning!!

File: docs/_pages/docs/03-understand/04-bonds.md [hunk 9]
```diff
@@ -2,10 +2,54 @@
...
+toc_sticky: true
 sidebar:
   title: '<img id="side-icon-verybig" src="/assets/vector/ticket-simple.svg"/>Bonds'
   nav: docs
 src: "_pages/docs/03-understand/04-bonds.md"
 ---
 
-{% include wip %}
\ No newline at end of file
+The fidelity bond is a small deposit that the user "locks" which will be relinquished after the trade is completed; however, users can lose their bond if they fail to follow the obligations of the contract.
```
[the remark this answers is not in the document]
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:13]: Agreed, updated accordingly

File: docs/_pages/docs/03-understand/04-bonds.md (hunk 2 above)
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:15]: Yes definitely agree. Did you have a recommended link? I have stumbled across the two sources below (it seems I'm not yielding many search results for Lightning hold invoice docs):

https://docs.lightning.engineering/the-lightning-network/payment-lifecycle
https://lightningwiki.net/index.php/HODL_Invoice

File: docs/_pages/docs/03-understand/04-bonds.md (hunk 1 above)
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:37]: Agreed, I like bullet points

Would a fifth condition for losing the bond be a peer trying to unilaterally cancel the order? Or is this perceived the same as trying to cheat or deceive peer

File: docs/_pages/docs/03-understand/04-bonds.md (hunk 2 above)
Comment (Reckless-Satoshi) [reply docs/_pages/docs/03-understand/04-bonds.md:15]: Those are good resources! Any work to me, I also like this one https://github.com/lightningnetwork/lnd/pull/2022

File: docs/_pages/docs/03-understand/04-bonds.md (hunk 1 above)
Comment (okada-blue) [reply docs/_pages/docs/03-understand/04-bonds.md:37]: I've gone ahead and thrown in the 5th bullet point just to make clear that cancelling uncollaboratively will result in lost bond

**Comment by Reckless-Satoshi** [comment]
Ready to merge! Will wait first for your invoice.

**Review by Reckless-Satoshi — APPROVED** [review]

**Comment by okada-blue** [comment]
> Ready to merge! Will wait first for your invoice.

Thank you!!
lnbc200u1p3j3ac2pp5yzdmgn23wwesn2x4lzm57vyfwf3ssmzhrcdfvmulsnvf6mmxun4sdqu2askcmr9wssx7e3q2dshgmmndp5scqzpgxqyz5vqsp5p574adek60z47qq2a9yse5u00wee7f0f0ngfhqrtjct5qskc7tus9qyyssq47jrt0sv6laum4cutckq9klngpdq00z7fcphrtfmh7m8gvwn8uxyn9fnnrem53e3lf09xphsdrq20v0z4y4dtv5w6nhkqa9jzs0a80qpvpz2yu

**Comment by Reckless-Satoshi** [comment]
> > Ready to merge! Will wait first for your invoice.
> 
> Thank you!! lnbc200u1p3j3ac2pp5yzdmgn23wwesn2x4lzm57vyfwf3ssmzhrcdfvmulsnvf6mmxun4sdqu2askcmr9wssx7e3q2dshgmmndp5scqzpgxqyz5vqsp5p574adek60z47qq2a9yse5u00wee7f0f0ngfhqrtjct5qskc7tus9qyyssq47jrt0sv6laum4cutckq9klngpdq00z7fcphrtfmh7m8gvwn8uxyn9fnnrem53e3lf09xphsdrq20v0z4y4dtv5w6nhkqa9jzs0a80qpvpz2yu

d250867fc0de5be979beadb24e24ddb6567d57c7882039d6930e5296a065c60a

**Pull request merged**